content/en/docs/concepts/workloads/_index.md

```diff
@@ -8,31 +8,53 @@ no_list: true
-a critical failure on the {{< glossary_tooltip text="node" term_id="node" >}} where that
-Pod is running means that all the Pods on that node fail. Kubernetes treats that level
-of failure as final: you would need to create a new Pod even if the node later recovers.
+`Pods` have a [defined lifecycle](/docs/concepts/workloads/pods/pod-lifecycle/).
+For example, once a `Pod` is running in your cluster then a critical fault on the
+{{< glossary_tooltip text="node" term_id="node" >}} where that `Pod` is running means that
+all the `Pods` on that node fail. Kubernetes treats that level of failure as final: you
+would need to create a new `Pod`, even if the node later recovers.
 
-However, to make life considerably easier, you don't need to manage each Pod directly.
-Instead, you can use _workload resources_ that manage a set of Pods on your behalf.
+However, to make life considerably easier, you don't need to manage each `Pod` directly.
+Instead, you can use _workload resources_ that manage a set of `Pods` on your behalf.
 These resources configure {{< glossary_tooltip term_id="controller" text="controllers" >}}
-that make sure the right number of the right kind of Pod are running, to match the state
+that make sure the right number of the right kind of `Pod` are running, to match the state
 you specified.
 
-Those workload resources include:
+Kubernetes provides several built-in workload resources:
 
-* [Deployment](/docs/concepts/workloads/controllers/deployment/) and [ReplicaSet](/docs/concepts/workloads/controllers/replicaset/)
-  (replacing the legacy resource {{< glossary_tooltip text="ReplicationController" term_id="replication-controller" >}});
-* [StatefulSet](/docs/concepts/workloads/controllers/statefulset/);
-* [DaemonSet](/docs/concepts/workloads/controllers/daemonset/) for running Pods that provide
-  node-local facilities, such as a storage driver or network plugin;
-* [Job](/docs/concepts/workloads/controllers/job/) and
-  [CronJob](/docs/concepts/workloads/controllers/cron-jobs/)
-  for tasks that run to completion.
+* [`Deployment`](/docs/concepts/workloads/controllers/deployment/) and [`ReplicaSet`](/docs/concepts/workloads/controllers/replicaset/)
+  (replacing the legacy resource
+  {{< glossary_tooltip text="ReplicationController" term_id="replication-controller" >}}).
+  `Deployment` is a good fit for managing a stateless application workload on your cluster,
+  where any `Pod` in the `Deployment` is interchangeable and can be replaced if needed.
+* [`StatefulSet`](/docs/concepts/workloads/controllers/statefulset/) lets you
+  run one or more related Pods that do track state somehow. For example, if your workload
+  records data persistently, you can run a `StatefulSet` that matches each `Pod` with a
+  [`PersistentVolume`](/docs/concepts/storage/persistent-volumes/), and replicate both
+  the pods and their storage across different failure zones. Your code, running in the `Pods`,
```

Comment:
This introduction has gone too far from the core concept of StatefulSet. Multi-zone deployment is not limited to stateful apps. It is a different topic from stateful.